### PR TITLE
Include SC20 BoF page

### PR DESCRIPTION
**Pull request by jeanbez**
Includes a new page for SC20 BoF based on previous pages. I already updated the date, time, and venue information with data from SC website.

### CODE REVIEW

**Comment by jeanbez** [comment]
I've gone ahead and created the template for the page based on previous versions. We just need to include the new information Jay mentioned.

**Review by gmarkomanolis — DISMISSED** [review]

**Review by johnbent — DISMISSED** [review]

File: templates/Pages/bof_sc20.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    </table>
+
+    <p>
+        Link to the <a class="link" href="https://sc20.supercomputing.org/presentation/?id=bof168&sess=sess313" target="_blank">official announcement</a>.
+    </p>
+
+    <h3>Abstract</h3>
+
+    <p>
+        The IO500 is quickly becoming the de facto benchmarking standard for HPC storage. Developed two years ago, the IO500 has published five official lists so far. A BoF highlight is the presentation of the seventh IO500 list.
```
Comment (adilger) [inline templates/Pages/bof_sc20.php:51]: There are six previous lists now.

**Review by adilger — CHANGES_REQUESTED** [review]
Need to update "six official lists so far".

**Comment by JulianKunkel** [comment]
Need to wait until after the weekend. Seems that SC releases the infos over the weekend.
As any attendee shall register to SC (as Exhibitor, which is free) to get access, maybe we shall not expose any URL at all.

**Review by gflofst — DISMISSED** [review]

**Review by JulianKunkel — DISMISSED** [review]

File: templates/Pages/bof_sc20.php (hunk 1 above)
Comment (jeanbez) [reply templates/Pages/bof_sc20.php]: Solved!

**Comment by jeanbez** [comment]
Updated to six previous lists as requested

**Review by gmarkomanolis — APPROVED** [review]

**Review by gflofst — APPROVED** [review]

**Review by adilger — APPROVED** [review]

**Review by JulianKunkel — APPROVED** [review]

**Review by johnbent — APPROVED** [review]

**Pull request merged**